docs/utilities/jasix.mdx

```diff
@@ -0,0 +1,299 @@
+---
+title: JASIX
+---
+
+## Overview
+
+The JASIX index is aimed at providing TABIX like indexing capabilities for the Nirvana JSON output.
+
+## Creating the JASIX index
+
+The JASIX index (that comes in a .jsi) file is generated on-the-fly with Nirvana output. It can also be generated independently by running the Jasix command line utility on the JSON output file. Please note that the Jasix utility can only consume JSON files that follow the Nirvana JSON output format. The following code blocks demonstrate the help menu and index generating functionalities of Jasix.
```

Comment:
I think we need to decide with casing we would like to use. In this paragraph alone, we have JASIX and Jasix.